at_events_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 3.0.2
+- Fixed at client dependency issue
+- Updated dependencies
+
```

Comment:
Write the change that we have removed that developers want to know. For more clear info.

Reply:
- `SuncStratagy` has been removed as of deprecation.

Like this ☝🏻